charts/datastore/templates/service.yaml

```diff
@@ -14,15 +14,21 @@ spec:
   selector:
     product: {{ template "datastore.name" . }}  
   ports:
-    {{ if or (eq .Values.datastore.serverMode "PLAIN") (eq .Values.datastore.serverMode "AUTOTLS") }} # AUTOTLS mode requires port 80 for communicating with LetsEncrypt
+    {{- if (eq .Values.datastore.serverMode "AUTOTLS") }} # AUTOTLS mode requires port 80 for communicating with LetsEncrypt
     - protocol: TCP
       port: 80
-      targetPort: 80
-      name: {{ printf "%s-http" (include "datastore.name" .) }}
+      targetPort: {{ .Values.datastore.letsEncryptContainerlPort }}
+      name: {{ printf "%s-lets-encrypt" (include "datastore.name" .) }}
     {{ end }}
-    {{ if or (eq .Values.datastore.serverMode "TLS") (eq .Values.datastore.serverMode "AUTOTLS") }}
+    {{- if or (eq .Values.datastore.serverMode "TLS") (eq .Values.datastore.serverMode "AUTOTLS") }}
     - protocol: TCP
       port: 443
-      targetPort: 443
```

Comment:
why is this removed?

Reply:
it's not removed it's defined at the bottom in line 32

Reply:
the `targetPort` of the dop is the same on both modes, just the service port changes and not configurable